package.json

```diff
@@ -7,23 +7,25 @@
     "wiki",
     "markdown"
   ],
-  "homepage": "http://github.com/kylefarris/J2M.git",
+  "homepage": "http://github.com/karimatthews/J2M.git",
   "repository": {
     "type": "git",
-    "url": "http://github.com/kylefarris/J2M.git"
+    "url": "http://github.com/karimatthews/J2M.git"
   },
   "description": "JIRA to MarkDown text format converter.",
   "main": "index.js",
   "devDependencies": {
-    "chai": "^3.3.0"
+    "chai": "^3.5.0",
+    "mocha": "^5.2.0"
   },
   "scripts": {
-    "test": "make test"
+    "test": "mocha --timeout 5000 --check-leaks --reporter spec test/*.js"
```

Comment:
In order to make tests running, this would require to install mocha globally, something like `npm install mocha -g`?
Or have I missed something?
probably just do `./node_modules/mocha/bin/mocha ......`